src/main/java/de/hysky/skyblocker/utils/container/ContainerMatcher.java

```diff
@@ -0,0 +1,17 @@
+package de.hysky.skyblocker.utils.container;
+
+import net.minecraft.client.gui.screen.ingame.HandledScreen;
+import org.jetbrains.annotations.NotNull;
+
+public interface ContainerMatcher {
+    /**
+     * Tests if the given screen should be handled by this matcher.
+     * @return {@code true} if this matcher should apply to the given screen, {@code false} otherwise
+     */
+    boolean test(@NotNull HandledScreen<?> screen);
```

Comment:
This should be for all screens. We have some screens that need tooltips or slot texts, and I don't see why it wouldn't work on all screens.

Reply:
Done in [`d261bca`](https://github.com/SkyblockerMod/Skyblocker/pull/788/commits/d261bca2d0e42dd77f2f6fbd45a1aedc9f5d6ad2).